packages/capabilities/src/access.js

```diff
@@ -41,7 +41,7 @@ export const AuthorizationRequest = Schema.struct({
   /**
    * DID of the Account authorization is requested from.
    */
-  iss: Account,
+  iss: Account.optional(),
```

Comment:
The is now optional, as per the RFC.

In the case of OAuth login, we don't know the email address of the user until after they have authorized and we have issued the delegation and attestation on their behalf.